src/ECS/ECS.cpp

```diff
@@ -0,0 +1,179 @@
+#include <entt/entt.hpp>
+#include <SDL.h>
+
+#include "GameManager.h"
+
+#include "Rendering/Renderer.h"
+#include "Rendering/Texture.h"
+#include "Rendering/Camera.h"
+
+#include "Parsers/Gm1File.h"
+
+#include "System/Logger.h"
+#include "ECS.h"
+
+#include "Assets/AssetsStorage.h"
+
+using namespace Sourcehold::Rendering;
+using namespace Sourcehold::System;
+using namespace Sourcehold::Parsers;
+using namespace Sourcehold::Game;
+
+// Test of EnTT, copypasted from EnTT docs
+// It is just to check if pipeline works with new import
+
+namespace Sourcehold {
+    namespace ECS {
+        struct PositionComponent {
+            int x;
+            int y;
+        };
+    
+        struct EntityTypeComponent {
+            EntityType type;
+        };
+
+        struct AnimatedComponent {
+            bool animate;
+            int frameIndex;
+        };
+
```

Comment:
You can also add empty structs as a tag.
For example marking an entity as renderable.
```C++
struct Renderable{
};
```
Something you might find useful later, perhaps.

Reply:
Will be usefull!